plugins/default/src/Ide/Plugin/Eval.hs

```diff
@@ -1,12 +1,6 @@
-{-# LANGUAGE DeriveAnyClass        #-}
-{-# LANGUAGE DeriveGeneric         #-}
-{-# LANGUAGE DuplicateRecordFields #-}
-{-# LANGUAGE LambdaCase            #-}
-{-# LANGUAGE NamedFieldPuns        #-}
-{-# LANGUAGE OverloadedStrings     #-}
-{-# LANGUAGE RecordWildCards       #-}
-{-# LANGUAGE ScopedTypeVariables   #-}
-{-# LANGUAGE TupleSections         #-}
+{-# LANGUAGE DeriveAnyClass, DeriveGeneric, DuplicateRecordFields #-}
+{-# LANGUAGE LambdaCase, NamedFieldPuns, OverloadedStrings        #-}
+{-# LANGUAGE RecordWildCards, ScopedTypeVariables, TupleSections  #-}
```

Comment:
May I ask why you change the style for this? It looks inconsistent with other modules...

Reply:
Oops, the LANGUAGE pragmas got squashed after I temporarily switched formatter to stylish-haskell, to get import lists reordered.
I will fix this.

Is there any *standard* set of code-formatter and configuration for HLS which can be used in handy?